DependencyInjection/LexikJWTAuthenticationExtension.php

```diff
@@ -223,7 +224,8 @@ private function processWithWebTokenConfig(array $config, ContainerBuilder $cont
             }
         }
         if ($config['access_token_verification']['enabled'] === true) {
-            $loader->load('web_token_verification.xml');
+            $phpLoader = new PhpFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
```

Comment:
I think we should migrate the whole configuration at once if we want to change it in the bundle.